stac_api/runtime/src/core.py

```diff
@@ -138,24 +138,27 @@ async def _search_base(
         result = await _super._search_base(search_request, **kwargs)
 
         if len(result["features"]) > 0:
```

Comment:
Do we even need the if conditional if we're using `try`

Reply:
I guess not but we could clean that up later